src/Messaging/src/RabbitMQ/Attributes/DeclareQueueBase.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Steeltoe.Messaging.Rabbit.Attributes
+{
+    public abstract class DeclareQueueBase : Attribute
+    {
+        public virtual string Durable { get; set; } = string.Empty;
```

Comment:
Some of these properties look like booleans - can/should we use an enum or constants instead of strings if we can't use `bool`?

Reply:
In many cases like this .. SPEL can be used instead of just a constant like "True" or "False" .. so to be functionally compat with the spring code .. it needs to remain